src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -5,12 +5,46 @@ import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
 import { Switch } from '../shared/Switch/Switch';
+import { FieldGroup, FieldItem, NumberField, SubmitButton } from '../form-helpers/form-helpers';
+import { Formik, Form } from 'formik';
 
 export const ContactPolicy = () => {
   const [active, toggleActive] = useState(false);
+  const [formSubmitted, setFormSubmitted] = useState(false);
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
+  const fieldNames = {
+    amount: 'amount',
+    interval: 'interval',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    initialValues[fieldNames.amount] = '';
+    initialValues[fieldNames.interval] = '';
+
+    return initialValues;
+  };
+
+  //TODO: refactor next block. (just to simulate form submission)
+  //BEGIN BLOCK
+  const delay = (ms) => new Promise((res) => setTimeout(res, ms));
```

Comment:
When you need to create a timeout, you can use useTimeout hook. 
PR: https://github.com/FromDoppler/doppler-webapp/pull/1428

This is to avoid console warnings indicating that all subscriptions must be canceled.